### DISCUSSION TITLE
Issues with wh/mi and efficiency calculations for new MIC LR AWD

### DISCUSSION THREAD

**Discussion by drewpost** [teslamate-org/teslamate#1455] [Q&A]
**Describe the bug**

I took delivery of a new Made in China 2021 LR AWD on Monday in the UK. This replaced my 2019 Fremont SR+. With the SR+, when looking at Drives, the fields for "kWh", "Consumption" and "Efficiency" always had values. With my new LR AWD, I occasionally get an efficiency calculation but I am not getting kWh or Consumption calculations being shown. They don't show in the table or in the drive details page. 

**Expected behavior**

Well all I've done is drive the LR AWD MIC Model 3. This produces the results described above every time. 

**Relevant entries from the logs**

```
Logs in text file as they were too long for me to be able to paste directly into the comment:
[teslamate_logs.zip](https://github.com/adriankumpf/teslamate/files/6155033/teslamate_logs.zip)

```

**Screenshots**

Behaviour with 2019 Fremont SR+ (expected)
<img width="1394" alt="Screenshot 2021-03-17 at 08 51 30" src="https://user-images.githubusercontent.com/7509768/111440336-24136c80-86fe-11eb-93d3-4a602da3787e.png">

Behaviour with new 2021 MIC LR AWD (Abnormal)
<img width="1444" alt="Screenshot 2021-03-17 at 08 47 51" src="https://user-images.githubusercontent.com/7509768/111440108-edd5ed00-86fd-11eb-8a3f-b30a6ce0a6f9.png">

Drive details page with abnormal behaviour
<img width="1416" alt="image" src="https://user-images.githubusercontent.com/7509768/111440802-9c7a2d80-86fe-11eb-8a1b-a0dfcef37ea9.png">

**Data**

Download Drive CSV from here:
https://1drv.ms/u/s!Asv5Qv5CeVBihIk-F2tdPYHeXqVj1w?e=qar3fW

**Environment**

- TeslaMate version: 1.21.6
- Type of installation: Docker
- OS TeslaMate is installed on: Raspberry Pi
- User OS & Browser: macOS 11.3 beta, Safari 14.1

**Comment by adriankumpf** [comment]
That should answer your question: [FAQ – Why are no consumption values displayed in Grafana?](https://docs.teslamate.org/docs/faq/#why-are-no-consumption-values-displayed-in-grafana)

I'll move this issue to the Q&A category.
